nri-kafka/src/Kafka/Worker/Internal.hs

```diff
@@ -108,10 +108,10 @@ subscription topic callback =
 subscriptionManageOwnOffsets ::
   (Aeson.FromJSON msg, Aeson.ToJSON msg) =>
   Text ->
-  Task Text (Dict.Dict Int Int) ->
+  ([OffsetParams] -> Task Text (Dict.Dict Int Int)) ->
```

Comment:
This looks fine to me, but also I'm wondering if we're asking library users to do more work for a performance improvement that's probably not going to make much difference (considering the number of offsets will always be a lowish number).

Reply:
🚨over-engineering🚨 perhaps the destination database will determine where to pull / put the data based on partition id?

agree we should drop it until it becomes an issue.